manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,187 @@
+        path: conditions
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.conditions
+      - description: The observed version of the vm import deployment.
+        displayName: Observed vm import Version
+        path: observedVersion
+        x-descriptors:
+        - urn:alm:descriptor:text
+      - description: The targeted version of the vm import deployment.
+        displayName: Target vm import Version
+        path: targetVersion
+        x-descriptors:
+        - urn:alm:descriptor:text
+      - description: The version of the vm import Operator
+        displayName: Vm import Operator Version
+        path: operatorVersion
+        x-descriptors:
+        - urn:alm:descriptor:text
+      version: v1beta1
+  description: VM import operator provides ability to import virtual machines from other infrastructure like oVirt/RHV
+  displayName: VM import operator
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - events
+          - configmaps
+          - secrets
+          - serviceaccounts
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - deployments/finalizers
+          verbs:
+          - '*'
+        - apiGroups:
+          - v2v.kubevirt.io
+          resources:
+          - vmimportconfigs
+          - vmimportconfigs/finalizers
+          - vmimportconfigs/status
+          verbs:
+          - '*'
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - '*'
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          - clusterroles
+          verbs:
+          - '*'
+        serviceAccountName: vm-import-operator
+      deployments:
+      - name: vm-import-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: vm-import-operator
+              operator.v2v.kubevirt.io: ""
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: vm-import-operator
+                operator.v2v.kubevirt.io: ""
+            spec:
+              containers:
+              - env:
+                - name: DEPLOY_CLUSTER_RESOURCES
+                  value: "true"
+                - name: OPERATOR_VERSION
+                  value: latest
+                - name: CONTROLLER_IMAGE
+                  value: quay.io/omachace/vm-import-controller:latest
+                - name: PULL_POLICY
+                  value: Always
+                - name: WATCH_NAMESPACE
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                image: quay.io/omachace/vm-import-operator:latest
```

Comment:
Please replace with quay.io/kubevirt/vm-import-operator:v0.2.0

Reply:
Thanks, for finding it I wonder, why the setup script didn't replace it...